.eslint-bin/pre-commit-hook.js

```diff
@@ -19,7 +19,17 @@ function getFilesModified(callback) {
 
     listFilesModified = status.files
       .filter(excludeNonCommitedFiles)
-      .map((file) => file.path)
+      .map((file) => {
+        if (file.index === 'R') {
+          // NOTICE: file path equals "old/path.js -> new/path.js" for renames
+          const matches = file.path.match(/^(?:.* -> )?(.*)$/);
+
+          return matches[1];
+        }
+
+        return file.path;
+      })
+      .filter(Boolean)
```

Comment:
What is it used for?
Does it fix another use case?

I can't see this in our other repositories (https://github.com/ForestAdmin/forestadmin-server/commit/df1442a2ec1717c96be3b8e3bdcb11e1f2325029)

Reply:
I don't really think that it's usefull in real-life situations, but I wanted to be sure that if the regexp did not capture the file path, an `undefined` value is not added to the final array.

Do you prefer that I remove this line?

Reply:
I answered my own question and removed the test.